indra/llcommon/llapr.cpp

```diff
@@ -606,7 +606,11 @@ S32 LLAPRFile::writeEx(const std::string& filename, const void *buf, S32 offset,
         apr_status_t s = apr_file_write(file_handle, buf, &bytes_written);
         if (s != APR_SUCCESS)
         {
-            LL_WARNS("APR") << " Attempting to write filename: " << filename << LL_ENDL;
```

Comment:
That seems really ill advised, why crash viewer when disk is out of space whne viewer can keep working? It should instead notify user, may be clear a bit of cache. Switching to read only mode might be an option too.

Reply:
And the biggest issue: LL_ERRS is a silent crash Without warning user, user is the only one in position to fix the issue, so user should be informed, preferably at startup, not runtime.

Reply:
Well, this will appear in the log. But the user can just keep trying running it over and over - there should definitely be some more explicit way for them to know about the problem.